### PR TITLE
removed classes unittest.TestCase

### LINKED ISSUE #3605: Convert bodhi/tests/messages/schemas/test_update.py to BaseTestCase
[bodhi/tests/messages/schemas/test_update.py](https://github.com/fedora-infra/bodhi/blob/develop/bodhi/tests/messages/schemas/test_update.py) still contains classes that derive from unittest.TestCase. We use pytest, so they could be migrated to derive from bodhi.tests.server.base.BaseTestCase.

This also allows the use of the simpler assert constructs instead of the self.assert*() methods.

The following PR is an example of the changes that might need to be done to this file:
#3574

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <subhamkumarrai03@gmail.com>

Fixes: #3605

### CODE REVIEW

File: bodhi/tests/messages/schemas/test_update.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 from bodhi.tests.messages.utils import check_message
 
 
-class UpdateMessageTests(unittest.TestCase):
+class UpdateMessageTests:
```
Comment (nphilipp) [inline bodhi/tests/messages/schemas/test_update.py]: In order to be recognized as a test, the class name has to start with `Test`. Would you rename it to `TestUpdateMessage`? Thanks!

**Review by nphilipp — CHANGES_REQUESTED** [review]

File: bodhi/tests/messages/schemas/test_update.py (hunk 1 above)
Comment (subhamkrai) [reply bodhi/tests/messages/schemas/test_update.py]: Yes sure I'll

File: bodhi/tests/messages/schemas/test_update.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 """Unit tests for the update message schemas."""
 
-import unittest
+
```
Comment (nphilipp) [inline bodhi/tests/messages/schemas/test_update.py:18]: I don't know why I didn't spot this earlier: this additional empty line makes flake8 complain (which is run during our automatic tests) -- there need to be two blank lines between the docstring of a module and the imports, this makes it three. Would you please remove it? Thanks!

**Review by nphilipp — CHANGES_REQUESTED** [review]

**Review by nphilipp — APPROVED** [review]
LGTM, thanks!

**Pull request merged**